packages/superset-ui-chart/src/types/Operator.ts

```diff
@@ -0,0 +1,43 @@
+/** List of operators that do not require another operand */
+export const UNARY_OPERATORS = ['IS NOT NULL', 'IS NULL'] as const;
```

Comment:
do we need to export `UNARY_OPERATORS` etc? seems there is no other place to use them.

Reply:
Not being used at the moment. I will remove `export`